_pages/en/strategy-open-first.md

```diff
@@ -0,0 +1,444 @@
+It must become digitally autonomous in its decisions while leveraging all possible options devising its solutions.
+
+### Implementing the Policy on Service and Digital
+
+Crucially, the open first strategy helps the ESDC achieve the high-level objectives of the Policy on Service and Digital:
+
+- Client service experience and government operations are improved through digital transformation approaches.
+- Guided by a commitment to the guiding principles and best practices of the GC Digital Standards: design with users; iterate and improve frequently; work in the open by default; use open standards and solutions; address security and privacy risks; build in accessible from the start; empower staff to deliver better services; be good data stewards; design ethical services; collaborate widely.
+
+The benefits of using open standards and OSS (the code or specification is public, free to reuse and collaboratively adapt and improve; it can be inspected for security issues, is independent from vendors, and gravitates towards interoperable systems) clearly underpin these objectives.
+
+The Policy on Service and Digital makes it clear that working in the open and collaborative working methods will be the norm within the GC's IT community to foster the sharing of code, data and solutions.
+The principal working methods encouraged by this open first strategy are open, inclusive and co-creative.
+
+To drive efficiencies across the GC, the Policy on Service and Digital hinges on the existence of the reusable solutions platform:
+
+ORE ?
+
+> The platform will be built around a catalogue containing reusable building blocks and services, open-source solutions, a corporate architecture, standards, best practices, support and consultancyservices.
+
+We have here a direct link between the Policy on Service and Digital and the open first strategy: the reusable solutions platform proves the benefits of a working culture based on the principles of open standards and OSS.
+
+Another key instrument in the Policy on Service and Digital is the Connecting Europe Facility (CEF);
+here too, the open first strategy provides an actionable link, able to take on board the lessons learned from CEF building blocks to build and foster communities around open source solutions.
+
+The Policy on Service and Digital refers to the vulnerability scanning of digital solutions and services, securing building blocks and patching procedures.
+OSS code can be inspected in detail and is easily included in systematic vulnerability scanning.
+This is another practical link between the Policy on Service and Digital and the open first strategy.
+
+Notes:
+
+Canada has established clear vision and designed it policies to nudge departments towards it.
+By adopting this strategy, ESDC is taking another step in realizing the vision outlined by the governement.
+
+### Sharing and reuse benefits all
```

Comment:
Economies of scale, working in the open leads to not only the product being reusable but also increase possibilities of collaborations on software development and overall pool of people continuously improving projects, whether internal or external to the organization. Benefits everyone to work in the open.

Reply:
This is more practical in the economics sense of the contributions models brought in by OSS and its associated methodologies.